code/game/objects/items/clown_items.dm

```diff
@@ -185,11 +185,11 @@
 	throw_range = 7
 	attack_verb_continuous = list("HONKS")
 	attack_verb_simple = list("HONK")
-	var/soundfile = 'sound/items/bikehorn.ogg'
 
 /obj/item/bikehorn/Initialize()
 	. = ..()
-	AddComponent(/datum/component/squeak, list(soundfile=1), 50, falloff_exponent = 20) //die off quick please)
+	LoadComponent(/datum/component/squeak, list('sound/items/bikehorn.ogg'=1), 50, 0, 0, 0, 0, 20, 0)
+	//LoadComponent so child types dont stack squeak components
```

Comment:
Put this comment before the LoadComponent.

Reply:
done